### DISCUSSION TITLE
I love the idea. Please go ahead

### DISCUSSION THREAD

**Discussion by sirlordt** [alshdavid/TypeScriptBC#82] [Ideas]
Yes yes yes finally!!!!. I love the idea please please go ahead!!!

**Comment by biels** [comment]
Same feeling. Please go ahead!!!

**Comment by alshdavid** [comment]
Thanks for the support! 😄 🍻

**Comment by Interpause** [comment]
I also think this sounds amazing, coming from someone who interested in learning Rust but have yet to start

**Comment by alshdavid** [comment]
That's great to hear, when you get some time to dig in and you're the kind of person who like video lectures, I highly recommend these resources:

https://youtu.be/a8abW3RlOn8
https://youtu.be/79phqVpE7cU
https://youtu.be/8M0QfLUDaaA
https://youtu.be/4DqP57BHaXI

**Comment by shortercode** [comment]
I've been working on my own ideas for a language inspired by TS and Rust for a couple of years ( sadly slowed down by finding the time/energy to work on it during my evenings ). It's very exciting to see more activity in the space. Particularly it looks like you have a lot more community involvement so there's a better chance of it getting completed sometime this decade! Happy to help out with the project, I'm sure there's some analogous decisions I made that could assist with the design.

**Comment by biels** [comment]
Also, maybe some ideas can be extracted from https://github.com/AssemblyScript/assemblyscript in the sense of making a lower level typescript

**Comment by mindplay-dk** [comment]
Not really an *issue*, but I feel that sentiment! 😄

@alshdavid time to open the discussions tab, so we can rave about how great this is going to be? 😏
